wsgi.py

```diff
@@ -144,11 +145,16 @@ def handler(event, context):
         u'headers': dict(new_headers)
     }
 
-    if response.data:
-        if isinstance(response.data, bytes):
-            returndict['body'] = base64.b64encode(response.data).decode('utf-8')
-            returndict["isBase64Encoded"] = "true"
+    data = response.data
+    if data:
+        if isinstance(data, bytes):
+            mimetype = response.mimetype or 'text/plain'
+            if mimetype.startswith('text/') or mimetype in TEXT_MIME_TYPES:
+                returndict['body'] = data.decode('utf-8')
```

Comment:
I think we only need the decode step on Python 3. The test should be modified to include unicode chars in the response body...